code/sorting/radix_sort/rad_sort.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream.h>
+#include <conio.h>
+#define MAX 10
+
+class radixsort{
+    int arr[MAX],n;
+    public:
+    void getdata();
+    void showdata();
+    void sortLogic();
+};
+
+void radixsort :: getdata(){
+    cout<<"How many elements you require : ";
+    cin>>n;
+    for(int i=0;i<n;i++)
```

Comment:
If you use vectors, you don't have to ask for size, only elements, and can read/push_back until an EOF.